lib/GADS/Role/Presentation/Column/File.pm

```diff
@@ -0,0 +1,11 @@
+package GADS::Role::Presentation::Column::File;
+
+use Moo::Role;
+
+sub after_presentation
+{   my ($self, $return) = @_;
+
+    $return->{extra_values} = $self->extra_values;
```

Comment:
I think the part of this PR which involves rendering the allowed mime types in the front-end and then returning it would be better to all be done in the back end.